app/popup.tsx

```diff
@@ -110,6 +110,13 @@ export class Popup extends React.Component<POPUP_PROPS & DispatchProps, {}> {
     this.props.handlePaginationChange(newPageNo_rowSize, tabId, field)
   }
 
+  displayInterceptMessage = () => {
+   return Object.keys(this.props.interceptData).map( tabId =>
+     this.props.tabId === Number(tabId) && this.props.interceptData[this.props.tabId] === "INTERCEPT_SUCCESS" ?
+    <a id="success-msg">{this.props.interceptData[this.props.tabId]}</a> :

```

Comment:
Why this is a `<a>`.
No need for all these complications.
`message && <div>{message}</div>`